core/contracts/oracle/interfaces/VotingInterface.sol

```diff
@@ -38,50 +38,73 @@ contract VotingInterface {
     enum Phase { Commit, Reveal, NUM_PHASES_PLACEHOLDER }
 
     /**
-     * @notice Commit your vote for a price request for `identifier` at `time`.
-     * @dev (`identifier`, `time`) must correspond to a price request that's currently in the commit phase. `hash`
-     * should be the keccak256 hash of the price you want to vote for and a `int salt`. Commits can be changed.
+     * @notice Commit a vote for a price request for `identifier` at `time`.
```

Comment:
Likewise, could you just apply any changes you make to `Voting.sol` to this contract as well?